robusta_krr/formatters/table.py

```diff
@@ -47,15 +76,19 @@ def table(result: Result) -> Table:
         caption=f"{result.score} points - {result.score_letter}",
     )
 
+    cluster_count = len(set(item.object.cluster for item in result.scans))
+
     table.add_column("Number", justify="right", no_wrap=True)
-    table.add_column("Cluster", style="cyan")
+    if cluster_count > 1:
+        table.add_column("Cluster", style="cyan")
     table.add_column("Namespace", style="cyan")
     table.add_column("Name", style="cyan")
     table.add_column("Pods", style="cyan")
     table.add_column("Old Pods", style="cyan")
     table.add_column("Type", style="cyan")
     table.add_column("Container", style="cyan")
     for resource in ResourceType:
+        table.add_column(f"{resource.name} Diff")
```

Comment:
if you are only calculating the difference for requests and displaying it there isnt this column redundant?

Reply:
nvrmnd saw that the multiple pods its different